### PR TITLE
[Windows] Don't try to read numa node from SysFS

### PR DESCRIPTION
**Pull request by mrry**
This makes the behavior on Windows consistent with Mac OS X, and prevents a needless error message when using GPUs.

Partially addresses #5986.

### CODE REVIEW

**Review by gunan — APPROVED** [review]

**Comment by gunan** [comment]
Jenkins failed to fetch changes from github, retrying tests.
Jenkins, test this please.

File: tensorflow/stream_executor/cuda/cuda_gpu_executor.cc [hunk 1]
```diff
@@ -900,6 +900,9 @@ static int TryToReadNumaNode(const string &pci_bus_id, int device_ordinal) {
 #if defined(__APPLE__)
   LOG(INFO) << "OS X does not support NUMA - returning NUMA node zero";
   return 0;
+#elif defined(PLATFORM_WINDOWS)
+  // Windows support for NUMA is not currently implemented. Return node 0.
+  return 0;
 #else
```
Comment (gunan) [inline tensorflow/stream_executor/cuda/cuda_gpu_executor.cc]: Don't we need an `#endif` at the end of the file for this?

Comment (gunan) [inline tensorflow/stream_executor/cuda/cuda_gpu_executor.cc]: Nevermind, I finally found it :)

**Review by gunan — APPROVED** [review]

**Comment by mrry** [comment]
@tensorflow-jenkins test this please.

**Pull request merged**